pages/install/install.rst

```diff
@@ -14,149 +14,55 @@
 
    .. class:: lead
 
-      The following instructions detail how to install Cantera on a variety of platforms. We highly
-      recommend that all new users install the Python interface via Conda. If you want to use one of
-      the other interfaces, please select the installation instructions for your platform.
+      The following instructions detail how to install the different Cantera interfaces
+      on a variety of platforms. We highly recommend that all new users install the
+      Python interface via :ref:`Conda <sec-install-conda>`.
 
-.. container:: container
+Installing the Cantera Python Interface
+=======================================
 
-   .. row::
+- The easiest way to install the Cantera Python interface on all operating systems is by
+  using :ref:`Conda <sec-install-conda>`.
 
-      .. container:: col-12
+- Windows users can use
+  :ref:`MSI installer packages <sec-install-windows>`.
```

Comment:
In Cantera/Cantera#1154 the MSI installers are replaced with wheels. Assuming that we intend to merge that prior to 2.6, this will need to be updated.

Reply:
My aim with this was to focus on the organization of these docs and write something that was correct now, not to provide all of the necessary updates for the 2.6 release, which will involve a lot of other changes.

I don't actually fully know what the plan is for the wheel-based installation -- are we just going to suggest  using `pip`, or do we want to provide instructions for installing wheels directly?

(I think you meant Cantera/cantera#1158, right?)

Reply:
> (I think you meant [Cantera/cantera#1158](https://github.com/Cantera/cantera/pull/1158), right?)

Oops, yes, thank you. Fair enough about limiting the changes.

As far as I know, the instructions for installing wheels directly is the same as installing from `pip`, except that you specify the wheel filename you've downloaded instead of just `cantera`.